appveyor.yml

```diff
@@ -0,0 +1,21 @@
+# Test against this version of Node.js
+environment:
+  nodejs_version: "4.4.4"
```

Comment:
Can we text v4.x & v6.x?
(see https://www.appveyor.com/docs/lang/nodejs-iojs#testing-under-multiple-versions-of-node-js-or-io-js)